elastalert/loaders.py

```diff
@@ -173,8 +173,12 @@ def load(self, conf, args=None):
                     continue
                 if rule['name'] in names:
                     raise EAException('Duplicate rule named %s' % (rule['name']))
-            except EAException as e:
-                raise EAException('Error loading file %s: %s' % (rule_file, e))
+            except EAException as e: 
```

Comment:
As discussed, this is not required since we are going to populate the existing terms during first time rule execution only.